scss/modules/_rows.scss

```diff
@@ -6,6 +6,10 @@
 
 /// Row element styles
 /// @group Rows
+/// @example
+///   <div class="row">
```

Comment:
Really anal point: this shouldn't be indented - the 2-space indent actually comes through in the docs example.